docs/docs/cmd/entra/group/group-set.mdx

```diff
@@ -0,0 +1,95 @@
+import Global from '/docs/cmd/_global.mdx';
+
+# entra group set
+
+Updates a Microsoft Entra group
+
+## Usage
+
+```sh
+m365 entra group set [options]
+```
+
+## Alias
+
+```sh
+m365 aad group set [options]
+```
+
+## Options
+
+```md definition-list
+`-i, --id [id]`	
+: The ID of the Microsoft Entra ID group to update. Specify either `id` or `displayName` but not both.
+
+`-n, --displayName [displayName]`	
+: The display name of the Microsoft Entra ID group to update. Specify either `id` or `displayName` but not both.
+
+`--newDisplayName [newDisplayName]`	
+: The new display name of the Microsoft Entra ID group. The maximum length is 256 characters.
+
+`--description [description]`
+: The new description for the group.
+
+`--mailNickname [mailNickname]`
+: The new mail alias for the group (part before the @). Maximum length is 64 characters.
+
+`--ownerIds [ownerIds]`
+: Comma-separated list of IDs of Microsoft Entra users that will be the group owners.
+
+`--ownerUserNames [ownerUserNames]`
+: Comma-separated list of UPNs of Microsoft Entra users that will be the group owners.
+
+`--memberIds [memberIds]`
+: Comma-separated list of IDs of Microsoft Entra users that will be the group members.
+
+`--memberUserNames [memberUserNames]`
+: Comma-separated list of UPNs of Microsoft Entra users that will be the group members.
+
+`--visibility [visibility]`
+: Specifies the group join policy and group content visibility for Microsoft 365 groups. Possible values are: `Private` or `Public`. Specify only when targeting a Microsoft 365 group.
+```
+
+<Global />
+
+## Remarks
+
+The `visibility` option affects the behavior of the group.
+
+With the `Public` visibility:
+- Anyone can join the group without needing owner permission. 
+- Anyone can view the attributes of the group. 
+- Anyone can see the members of the group.
+
+With the `Private` visibilty:
+- Owner permission is needed to join the group.
```

Comment:
```suggestion
- Owner approval is needed to join the group.
```

Reply:
Copy/paste from the official doc, but `approval` makes more sense